mappings/net/minecraft/util/registry/RegistryTracker.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/class_5455 net/minecraft/util/registry/RegistryTracker
+	FIELD field_25918 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25919 INFOS Ljava/util/Map;
+	METHOD method_30518 getDimensionTypeRegistry ()Lnet/minecraft/class_2378;
+	METHOD method_30519 load (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5455$class_5457;
+		ARG 0 resourceManager
+	METHOD method_30520 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/MapCodec;Z)Lcom/google/common/collect/ImmutableMap$Builder;
+		ARG 0 infosBuilder
+		ARG 1 registryRef
+		ARG 2 elementCodec
+		ARG 3 synced
+	METHOD method_30524 addBuiltinEntries (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_2378;)V
+		ARG 0 tracker
+		ARG 1 registry
+	METHOD method_30525 setupBuiltin (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5321;)V
+		ARG 0 tracker
+		ARG 1 registryRef
+	METHOD method_30526 load (Lnet/minecraft/class_5382;Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5455$class_5456;)V
```

Comment:
what does this load and the other load method do differently?

Reply:
This loads one registry, the other loads all registries from a resource manager **on client**

Reply:
ALso the other one calls this one